types/configs/index.d.ts

```diff
@@ -1,7 +1,10 @@
+import { fromCallback } from 'cypress/types/bluebird';
```

Comment:
this is actually a problem as it makes the cypress dependency, and causing the following errors for the apps
```
Error: node_modules/@editorjs/editorjs/types/configs/index.d.ts:1:30 - error TS2307: Cannot find module 'cypress/types/bluebird' or its corresponding type declarations.

1 import { fromCallback } from 'cypress/types/bluebird';
```